pkg/kindsys/report.go

```diff
@@ -1,390 +0,0 @@
-//go:build ignore
```

Comment:
need to preserve this somewhere, and update what's in the Makefile to point to its new location.

wherever the `report.json` pops out, we'll also need to update the kinds dashboard on ops to point to the new location